### ISSUE TITLE
default URL location leads to broken links

### ISSUE THREAD

**Issue by wez** [qmx/cargo-docserver#1]
TL;DR: I think you need to add an explicit `/index.html` to the URL in this redirect:
https://github.com/qmx/cargo-docserver/blob/master/src/main.rs#L83

Steps to repro:

1. `cargo docserver`
2. open `http://0.0.0.0:4000` in the browser
3. The redirect sends me to `http://0.0.0.0:4000/cratename`
4. Click on a link to one of the modules within the crate
5. Browser 404s on a URL like `http://0.0.0.0:4000/modulename/index.html` (the `cratename` portion is missing)

If I open `http://0.0.0.0:4000/cratename/index.html` in the browser, the link for the module is `http://0.0.0.0:4000/cratename/modulename/index.html` and loads successfully.

**Comment by qmx** [comment]
I couldn't repro here, the crates I have here resolve to a flat namespace, so all my links here will end up as `http://localhost:4000/modulename vs /cratename/modulename`

Do you have an example repo for me to take a look? Happy to investigate more

**Comment by wez** [comment]
Yep, I realize my report was rather abstract.  Here's a repro:

```
git clone https://github.com/dcuddeback/termios-rs
cargo doc
cargo docserver
```

open `http://0.0.0.0:4000` in your browser and then scroll all the way down to the modules section, then click on the `ffi` module and you should see the 404

**Comment by qmx** [comment]
0.1.1 released with the fix, thanks for the bug report!
